view/adminhtml/web/vue/MenuType.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <fieldset class="admin__fieldset fieldset-wide">
+        <simple-field
+            :label="config.translation.nodeName"
+            id="node_name"
+            type="textarea"
+            v-model="item.title"
+        >
+        </simple-field>
+        <simple-field
+            :label="config.translation.nodeClasses"
+            id="node_classes"
+            type="text"
+            v-model="item.classes"
+        >
+        </simple-field>
+        <div class="admin__field field field-title">
+            <label
+                class="label admin__field-label"
+                for="node_type"
+            >
+                {{ config.translation.nodeType }}
+            </label>
+            <div class="admin__field-control control">
+                <select
```

Comment:
Why not `v-select`?

Reply:
because we have only few options, so will not need search filter or other v-select features

Reply:
But it will be easier to keep styles and navigation consistent. You can always disable search option for simple selects.